types.ts

```diff
@@ -970,6 +977,45 @@ export type VariousSupportedAuthentication =
  */
 export type VariousSupportedAuthenticationTypes = $Values<Pick<VariousSupportedAuthentication, 'type'>>;
 
+/**
+ * @hidden
+ */
+export enum ReservedAuthenticationKeys {
```

Comment:
Do we really need these? Built-in vs auxiliary auth types will be in separate parts of the manifest so I don't think there's any risk of collision.

Reply:
Ah I see, you need a way to reference these on the formulas.